### ISSUE TITLE
The truth table in REQ_null-predicate.adoc is about "NOT" but not about isNullPredicate

### ISSUE THREAD

**Issue by kunlinyu** [opengeospatial/ogcapi-features#970]
https://github.com/opengeospatial/ogcapi-features/blob/f3d0ce50676afddba1e61bc039050dac2ec85069/cql2/standard/requirements/basic-cql2/REQ_null-predicate.adoc?plain=1#L12

**Comment by cportele** [comment]
Yes, this looks like an error. I think the table 3 should be in section 6.2 along the clarification how to evaluate `AND` and `OR`. 

`/req/basic-cql2/null-predicate` should specify that `IS NULL` evaluates to `TRUE`, if the expression is `NULL` and otherwise evaluates to `FALSE`.

**Comment by pvretano** [comment]
@pvretano will review and patch as necessary.
